src/sentry/utils/outcomes.py

```diff
@@ -74,9 +74,8 @@ def track_outcome(
 
     outcomes_config = kafka_config.get_topic_definition(settings.KAFKA_OUTCOMES)
     billing_config = kafka_config.get_topic_definition(settings.KAFKA_OUTCOMES_BILLING)
-    use_billing = (
-        outcome.is_billing() and settings.KAFKA_TOPICS[settings.KAFKA_OUTCOMES_BILLING] is not None
-    )
+
+    use_billing = outcome.is_billing()
```

Comment:
Now if the outcome is a billing one, it always goes to the billing topic. There is no flipping the topic depending on seettings.

Reply:
Hi Lyn, does this mean that accepted and rate limited outcomes are only processed in billing topic, i.e. no other processes care about accepted and rate limited outcomes.

Reply:
This was always the case in prod since the cluster mapping for outcomes-billing topic was always set and this line evaluated to True. The thing that is changing is dev, which will now match prod. I added https://github.com/getsentry/snuba/pull/5603 to Snuba's devserver so we will ingest from both topics in dev.